docs/vm-images.md

```diff
@@ -50,12 +48,129 @@ settings:
     project: ubuntu-os-cloud
 ```
 
-[htcondor-configure]: ../community/modules/scheduler/htcondor-configure/README.md
-[nfs-server]: ../community/modules/file-system/nfs-server/README.md
-[DDN-EXAScaler]: ../community/modules/file-system/DDN-EXAScaler/README.md
-[exascalerimages]: https://github.com/DDNStorage/exascaler-cloud-terraform/blob/master/gcp/README.md#boot-image-options
-[omnia-install]: ../community/modules/scripts/omnia-install/README.md
-[hpc-slurm-ubuntu2004.yaml]: ../community/examples/hpc-slurm-ubuntu2004.yaml
+## Compatibility table
```

Comment:
![image](https://github.com/GoogleCloudPlatform/hpc-toolkit/assets/22925983/bedf4e4d-72ae-43bd-be1e-70636a4cc0c9)

Reply:
Sorry about that, made some copy-paste errors when I moved things around in the table!  It should be fixed now.